qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/PatientVisitsDashboardPage.java

```diff
@@ -26,7 +26,7 @@
 	private static final By TRANSFER_TO_WARD = By.id("referenceapplication.realTime.simpleTransfer");
 	private static final By EXIT_FROM_INPATIENT = By.id("referenceapplication.realTime.simpleDischarge");
 	private static final By ACTIONS_DROPDOWN = By.cssSelector("#content span.dropdown-name");
-	private static final By MERGE_VISITS = By.cssSelector("#content div.actions.dropdown ul li:nth-child(2) > a");
+	private static final By MERGE_VISITS = By.linkText("Merge Visits");
```

Comment:
any reason why you are changing the element locator?

Reply:
while testing out the test locally against the qaServer, that particular element locator was instead pointing to the `add past visit`.

Reply:
is there any legacy test that is failing as a result of the element locator?

Reply:
not really, gonna sort it in the nxt commit

Reply:
@kdaud look out for the logs at https://github.com/openmrs/openmrs-contrib-qaframework/runs/5107649752?check_suite_focus=true. 

So when i change the element as i had earlier done, the test passes that snag